Doc/library/profile.rst

```diff
@@ -151,11 +151,17 @@ might try the following sort calls::
    p.sort_stats('name')
    p.print_stats()
 
+   p.sort_stats(SortKey.name)
+   p.print_stats()
```

Comment:
Hello and thanks for the PR!

Here I would recomment keeping only the example with the enum, since it’s the way that we want to recommend.

To match the import style (see `import pstats` in the previous code block), the example would need to be `p.sort_stats(pstats.SortKey.NAME)` (assuming the upper-case fix asked by the other reviewer is done).

I wonder if the enum should be defined on the Stats class, to make code shorter: `p.sort_stats(p.SortKey.NAME)`.  Maybe worth asking on the ticket to get other people’s opinion.

Reply:
I would change the import to

    from pstats import Stats, SortKey

Reply:
The `pstats` module is all about stats processing, so there is no need to hide `SortKey` inside the `Stats` class.

Reply:
Thank you for the comment.   Re: 'Here I would recomment keeping only the example with the enum, since it’s the way that we want to recommend.'    I could do that, but I have also received comment about having both example as the string argument is still valid.   Showing only the enum would make the doc incomplete.

Reply:
Yes there is a need to also show an example with a string form, but not all code examples need both forms.

Reply:
@mwidjaja 

When you make the duplicate `SortKey`s actually duplicates, please put the better name first -- something like:

    class SortKey(str, Enum):
        calls = 'calls'
        ncalls = 'calls'
        cumulative = 'cumulative'
        cumtime = 'cumulative'
        module = 'module'
        file = 'module'
        filename = 'module'
        line = 'line'
        name = 'name'
        nfl = 'nfl'
        pcalls = 'pcalls'
        stdname = 'stdname'
        time = 'time'
        tottime = 'time'